spec/support/contexts/shared_rd_donation_value_context.rb

```diff
@@ -154,7 +154,7 @@ def generate_expected(donation_id, payment_id, charge_id, card, supporter, nonpr
   end
 
   def generate_expected_refund(data={})
-    result = {}.with_indifferent_access
+    result = {}.to_deprecated_h.with_indifferent_access
```

Comment:
should this be

`result = data.to_deprecated_h.with_indifferent_access`

Reply:
Turns out that code isn't actually used, I'm going to remove it in a separate PR

Reply:
#1017